src/pages/sns/faq.tsx

```diff
@@ -689,6 +698,64 @@ function SnsFaqPage() {
                 </Link>
               </ul>
             </Faq>
+          </FaqSection>
+
+          <FaqSection
+            id="treasury"
+            title={
+              <h2 className="tw-heading-3 text-gradient mb-12 md:mb-0 md:tw-heading-60">
+                SNS DAO treasury
+              </h2>
+            }
+          >
+            <Faq title="What is the SNS treasury?">
+              After a decentralization swap successfully concludes, the ICP acquired through the
+              swap is allocated to the SNS DAO's treasury. Optionally, the DAO can allocate a number of
+              SNS tokens to be in the DAO's treasury, which all teams so far have done. The SNS treasury
```

Comment:
```suggestion
              SNS tokens to be in the DAO's treasury, which all SNSs so far have done. The SNS treasury
```
(it is not really up to any teams, but rather the NNS decides, so I propose to say "SNS" or "DAO" instead )